### PR TITLE
Add team docker and config files and interfaces file

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: ansible/roles/sonicv2/tasks/main.yml [hunk 1]
```diff
@@ -8,7 +8,17 @@
             mode=0644
   register: if_copy
   tags: network,unsafe
-  when: sonic_version == "v2"
+
```
Comment (qiluo-msft) [inline ansible/roles/sonicv2/tasks/main.yml]: Why remove 'when' clause?

File: ansible/roles/sonicv2/tasks/main.yml [hunk 2]
```diff
@@ -8,7 +8,17 @@
...
+# Setup LAG Interfaces
+- name: Copy LAG Interfaces File
+  become: true
+  template: src=lag_interfaces.j2
+            dest=/etc/network/interfaces.d/lag_interfaces
+            owner=root
+            group=root
+            mode=0644
+  register: if_copy
```
Comment (qiluo-msft) [inline ansible/roles/sonicv2/tasks/main.yml]: Is it going to overwrite previous 'if_copy'?

File: ansible/roles/sonicv2/tasks/team_interface.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+          state=directory
+
+  - name: Copy teamd configuration file
+    become: true
+    template: src=teamd.j2
+              dest=/etc/teamd/{{ port_channel["name"] }}.conf
+              owner=root
+              group=root
+              mode=644
+
```
Comment (qiluo-msft) [inline ansible/roles/sonicv2/tasks/team_interface.yml]: reboot after template?

File: ansible/roles/sonicv2/tasks/main.yml (hunk 1 above)
Comment (stcheng) [inline ansible/roles/sonicv2/tasks/main.yml]: it is in sonicv2 folder so no need to have this

File: ansible/roles/sonicv2/tasks/main.yml [hunk 4]
```diff
@@ -33,3 +43,6 @@
 
 - include: quagga.yml
   tags: quagga,unsafe
+
+- include: team.yml
```
Comment (lguohan) [inline ansible/roles/sonicv2/tasks/main.yml]: >team [](start = 11, length = 4)

rename to teamd?

File: ansible/roles/sonicv2/tasks/team.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+- name: Start the team docker container
```
Comment (lguohan) [inline ansible/roles/sonicv2/tasks/team.yml]: teamd might be a better name.

File: ansible/roles/sonicv2/tasks/team.yml [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+  include: sonicdocker.yml
+  vars:
+    docker_container: team
```
Comment (lguohan) [inline ansible/roles/sonicv2/tasks/team.yml]: >team [](start = 22, length = 4)

rename to teamd?

**Comment by lguohan** [comment]
> 

can you provide minigraph examples?

---
Refers to: ansible/group_vars/sonic_latest/package_versions.yml:25 in 72618ef. [](commit_id = 72618ef6681962ac0faf326d0f4b702e85b9f585, deletion_comment = False)

File: ansible/roles/sonicv2/templates/lag_interfaces.j2 [hunk 7]
```diff
@@ -0,0 +1,19 @@
+#
+{% block banner %}
+# =========== Managed by Ansible DO NOT EDIT! ========================
+# generated by templates/interfaces.j2 using minigraph_facts.py
```
Comment (lguohan) [inline ansible/roles/sonicv2/templates/lag_interfaces.j2]: lag_interfaces.j2

File: ansible/roles/sonicv2/templates/lag_interfaces.j2 [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+# file: vlan_interfaces
+{% endblock %}
+#
+{% block lag_interface %}
+# The switch VLAN interfaces
```
Comment (lguohan) [inline ansible/roles/sonicv2/templates/lag_interfaces.j2]: >The switch VLAN interfaces [](start = 2, length = 26)

switch VLAN interfaces?

File: ansible/roles/sonicv2/templates/lag_interfaces.j2 [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+{% for interface in minigraph_portchannel_interfaces %}
+auto {{ interface['name'] }}
+iface {{ interface['name'] }} {{ 'inet' if interface['addr'] | ipv4 else 'inet6' }} static
+    bridge_ports {{ interface['members'] }}
+    address {{ interface['addr'] }}
+    netmask {{ interface['mask'] }}
+{% endfor %}
+{% endblock lag_interface %}
```
Comment (lguohan) [inline ansible/roles/sonicv2/templates/lag_interfaces.j2]: we need bridges_ports here? Is this for lag for vlan?

File: ansible/roles/sonicv2/tasks/main.yml (hunk 2 above)
Comment (lguohan) [inline ansible/roles/sonicv2/tasks/main.yml]: why do we seperate this file with the interfaces file?

**Comment by lguohan** [comment]
:clock1:

**Comment by stcheng** [comment]
close this and open #69.

**Pull request closed without merging**